okapi-common/src/main/java/org/folio/okapi/common/CqlUtil.java

```diff
@@ -65,8 +67,8 @@ public static CQLNode reducer(CQLNode vn1, CQLTermNode tn, Comparator<CQLTermNod
       } else {
         CQLSortNode sn = new CQLSortNode(n2);
         List<ModifierSet> mods = n1.getSortIndexes();
-        for (ModifierSet mSet : mods) {
-          sn.addSortIndex(mSet);
+        for (ModifierSet mset : mods) {
+          sn.addSortIndex(mset);
```

Comment:
This circumvents the intention of the rule to avoid the single letter abbreviation `m` in the first chunk of the two chunk name `mSet` or `mset`. It violates the rule that each word within the variable is camel cased. `mset` is not a word. Either rename to `set` or `modifierSet` or `m` or `s` or use `mods.forEach(sn::addSortIndex)`.

Reply:
Pedantic. I'm not gonna do it, buf if you make a PR, I'll probably approve